Makefile

```diff
@@ -13,6 +11,7 @@ endif
 
 ifeq ($(BUILD_LIBBPFGO),1)
 ifneq ($(shell command -pv $(CLANG)),)
```

Comment:
This line needs to be ($(shell command -pv $(CLANG);),) because on shells which have "command' as a built-in the following errors is produced:  `make: command: Command not found`, that's due to Make thinking this is a simple command and won't spawn a shell and will try to fork/exec it. The fix is to add a special shell character, like ';' which causes it to be run within a shell. I hit this on ubuntu 18.04 with make 4.1. For reference here is an explanation by make's maintainer himself: https://stackoverflow.com/questions/17547625/how-to-use-shell-builtin-function-from-a-makefile

Reply:
Ack